### PR TITLE
Form

### PR DESCRIPTION
**Pull request by AbbasSarwar**
### **Features**
* Heading
* Paragraph.
* Form.
* Validation inputs.

### Validations
* Through error if the input is empty.
* If the email doesn't have '@'.
* Textarea can have a maximum of 50 characters.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -205,22 +205,28 @@ <h2>Languages <span class="arrow-down"><i class="fa-solid fa-caret-down"></i></s
...
+                <ul>
+                    <li>
+                        <input type="text" name="Username" id="name" minlength="6" maxlength="30" required placeholder="Full Name">
+                    </li>
+                    <li>
+                        <input type="email" name="Email" id="mail" required placeholder="Email" pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$]">
+                    </li>
+                    <li>
+                        <textarea name="Message" id="msg" cols="30" rows="10" required placeholder="Write Your message here." maxlength="50"></textarea>
+                    </li>
```
Comment (divinecharlotte) [inline index.html]: - [x] kindly consider making the max length '500' to meet  the bellow requirement
> Message field accepts up to 500 chars.

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
# Hello @AbbasSarwar  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- GitHub Flow observed ✔️
- Good commit messages ✔️

## Required Changes ♻️

- [x] kindly consider adding a working link on tour MIT license to make your readme look more professional

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (AbbasSarwar) [reply index.html]: I have fixed it.

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @AbbasSarwar , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- Your commit messages are descriptive. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**